core/src/test/java/com/adobe/aem/commons/assetshare/components/structure/impl/HeaderImplTest.java

```diff
@@ -0,0 +1,35 @@
+package com.adobe.aem.commons.assetshare.components.structure.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import org.apache.sling.models.factory.ModelFactory;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import io.wcm.testing.mock.aem.junit.AemContext;
+
+public class HeaderImplTest {
+    
+    private static final String RESOURCEPATH = "/content/asset-share-commons/en/light";
```

Comment:
maybe not the sample content path, since there isn’t a real correlation between these 2 — this naming may confuse and imply there is. The integration test project will build out a tree under /content/asset-share-commons/test (tho perhaps we should include Lang copy prefix if that could aid in testing multi-Lang stuff down the road? Wdyt?)

Reply:
Just an example, I am fine to change that. In the end the path also doesn't matter.

Reply:
Agree, Doesn't technically matter, but consistency is next to cleanliness and you know what that's next to ;) ..